src/page-racial-disparities/PageRacialDisparities.js

```diff
@@ -0,0 +1,543 @@
+            {supervisionTypeText[supervisionType]}, rather than a crime),{" "}
+            <DynamicText>
+              {formatAsPct(supervisionTypeMetrics.absconsionRate)}
+            </DynamicText>{" "}
+            of the time for absconsion from{" "}
+            {supervisionTypeText[supervisionType]}, and{" "}
+            <DynamicText>
+              {formatAsPct(supervisionTypeMetrics.newOffenseRate)}
+            </DynamicText>{" "}
+            of the time for new crimes. In contrast, overall revocations for
+            technical violations are{" "}
+            <DynamicText>
+              {formatAsPct(supervisionTypeMetrics.technicalRateOverall)}
+            </DynamicText>
+            , revocations for absconsion{" "}
+            <DynamicText>
+              {formatAsPct(supervisionTypeMetrics.absconsionRateOverall)}
+            </DynamicText>{" "}
+            and revocations for new crime{" "}
+            <DynamicText>
+              {formatAsPct(supervisionTypeMetrics.newOffenseRateOverall)}
+            </DynamicText>
+            .
+          </p>
+        </>
+      ),
+    },
+    {
+      title: "Can programming help reduce disparities?",
+      description: (
+        <>
+          <p>
+            Programming is designed to improve outcomes for justice-involved
+            individuals. If programming is utilized more by groups
+            overrepresented in the justice system, it could help close the gap.
+          </p>
+          <p>
+            In 2018, North Dakota launched Free Through Recovery, a wrap-around
+            behavioral health program that helps those with behavioral health
+            challenges to succeed on community supervision.{" "}
+            <DynamicText>{sentenceCase(ethnonym)}</DynamicText> are{" "}
+            <DynamicText>{formatAsPct(metrics.ftrPopulationRate)}</DynamicText>{" "}
+            of referrals to FTR, a greater representation than their overall{" "}
+            <DynamicText>
+              {formatAsPct(
+                metrics.supervisionMetrics[TOTAL_KEY].populationRate
+              )}
+            </DynamicText>{" "}
+            of the supervision population.
+          </p>
+          <p>
+            On July 1, 2020, the DOCR began a pilot of pre-trial services,
+            programming designed to reduce the number of people held in jail
+            between arrest and appearance before a judge.{" "}
+            <DynamicText>{sentenceCase(sentenceCase(ethnonym))}</DynamicText>{" "}
+            make up{" "}
+            <DynamicText>
+              {formatAsPct(metrics.pretrialPopulationRate)}
+            </DynamicText>{" "}
+            of individuals enrolled in pre-trial services rather than detained
+            prior to appearance in court.
+          </p>
+        </>
+      ),
+    },
+    {
+      title:
+        "What are we doing to further improve disparities in criminal justice in North Dakota?",
```

Comment:
We use "reduce disparities" in other places on the page, and I think it reads more plainly than "improve disparities."

Reply:
another one for @kbinney

Reply:
and/or @hobuobi

Reply:
Fair enough!